src/main/java/mate/academy/internetshop/dao/jdbc/UserDaoJdbcImpl.java

```diff
@@ -0,0 +1,211 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import mate.academy.internetshop.dao.UserDao;
+import mate.academy.internetshop.exception.AuthenticationException;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Role;
+import mate.academy.internetshop.model.User;
+import org.apache.log4j.Logger;
+
+@Dao
+public class UserDaoJdbcImpl extends AbstractDao<User> implements UserDao {
+    private static Logger logger = Logger.getLogger(UserDaoJdbcImpl.class);
+
+    public UserDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    public Set<Role> addRoleToDB(User user) {
```

Comment:
```suggestion
    private Set<Role> addRoleToDB(User user) {
```
@TatianaPirch can this method be private?